explainer/index.bs

```diff
@@ -727,6 +727,87 @@ When using advanced methods to transfer data to the GPU (with a rolling list of
 
 ## Canvas Output ## {#canvas-output}
 
+Historically, drawing APIs (2d canvas, WebGL) are initialized from canvases using `getContext()`.
+However, WebGPU is more than a drawing API, and many applications do not need a canvas.
+WebGPU is initialized without a canvas - see [[#initialization]].
+
+Following this, WebGPU has no "default" drawing buffer.
+Instead, a WebGPU device may be connected to *arbitrarily many* canvases (via
+`GPUCanvasContext`s) and render to all of them.
```

Comment:
would be good to add that we can also draw to none of them

Reply:
Done